app/src/main/java/com/courseproject/tindar/entities/Message.java

```diff
@@ -0,0 +1,63 @@
+package com.courseproject.tindar.entities;
+
+import java.sql.Timestamp;
+
+public class Message {
+    private String messageID;
+    private String text;
+    private Timestamp timestamp;
+    private String sentFrom;
+    private String sentTo;
+    private String conversationId;
+
+    // Getter methods
+    public String getMessageID() {
+        return messageID;
+    }
+
+    public String getText() {
```

Comment:
rename to getMessageContent so that supporting gifs, images, etc. doesn't become a pain later.